units/patch_intent.go

```diff
@@ -1026,3 +1016,17 @@ func (j *patchIntentProcessor) sendGitHubErrorStatus(patchDoc *patch.Patch) {
 
 	j.AddError(update.Error())
 }
+
+// sendGitHubStatusMessage sends a successful status to Github with the given message.
+func (j *patchIntentProcessor) sendGitHubStatusMessage(patchDoc *patch.Patch, msg string) {
```

Comment:
Can we call this `sendGitHubSuccessMessage` or something, since it only means success?